src/AppSearch/Request/GetCountAnalytics.php

```diff
@@ -33,6 +35,7 @@ public function __construct(string $engineName)
 	{
 		$this->method = 'GET';
 		$engine_name = urlencode($engineName);
+		$this->headers['Content-Type'] = 'application/json';
```

Comment:
Is this content type affected by the compatibility header?
https://github.com/elastic/clients-team/issues/389

Reply:
No, the compatibility header is only for Elasticsearch.